src/Stratis.Bitcoin.Features.Miner/Controllers/MinerController.cs

```diff
@@ -118,5 +115,17 @@ public IActionResult StartStaking([FromBody]StartStakingRequest request)
                 return ErrorHelpers.BuildErrorResponse(HttpStatusCode.BadRequest, e.Message, e.ToString());
             }
         }
+
+        private Wallet.Wallet GetWallet(IWalletManager walletManager, StartStakingRequest request)
```

Comment:
please no new code outside tests without xmldoc

Reply:
Ah @Neurosploit you've been caught